pages/projects/[id]/index.js

```diff
@@ -3,7 +3,7 @@ import useSWR from "swr";
 
 import Error from "@/components/Error";
 import Heading from "@/components/Heading";
-import Button from "@/components/Button";
+import BackLink, { DeleteButton } from "@/components/Buttons";
```

Comment:
Why is one import the default one? You can have a module without defaults so you can safely make all of them the same ^^